<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>production_reports</class>
 <widget class="QMainWindow" name="production_reports">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>656</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reports</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../home/src/res.qrc">
    <normaloff>:/images/Printer.png</normaloff>:/images/Printer.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Daily Reel Usage Analysis</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Start Date:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="dtpStartDate">
            <property name="displayFormat">
             <string>dd MMM yyyy</string>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>End Date:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="dtpEndDate">
            <property name="displayFormat">
             <string>dd MMM yyyy</string>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="cmdShow">
            <property name="text">
             <string>Show</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextBrowser" name="txtResult"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Corrugator Data Entry</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Start Date:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="dtpStartDate_2">
            <property name="displayFormat">
             <string>dd MMM yyyy</string>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>End Date:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="dtpEndDate_2">
            <property name="displayFormat">
             <string>dd MMM yyyy</string>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="cmdShow_2">
            <property name="text">
             <string>Show</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextBrowser" name="txtCorrugatorBrowser"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../home/src/res.qrc"/>
  <include location="../../home/src/res.qrc"/>
 </resources>
 <connections/>
</ui>
